contracts/minitoken/solidity/migrations/2_token_migration.js

```diff
@@ -1,61 +1,41 @@
 const MiniToken = artifacts.require("MiniToken");
-const IBCHost = artifacts.require("@hyperledger-labs/yui-ibc-solidity/IBCHost");
 const IBCClient = artifacts.require("@hyperledger-labs/yui-ibc-solidity/IBCClient");
 const IBCConnection = artifacts.require("@hyperledger-labs/yui-ibc-solidity/IBCConnection");
-const IBCChannel = artifacts.require("@hyperledger-labs/yui-ibc-solidity/IBCChannel");
-const IBCHandler = artifacts.require("@hyperledger-labs/yui-ibc-solidity/IBCHandler");
+const IBCChannel = artifacts.require("@hyperledger-labs/yui-ibc-solidity/Channel");
+const IBCChannelHandshake = artifacts.require("@hyperledger-labs/yui-ibc-solidity/IBCChannelHandshake");
+const IBCPacket = artifacts.require("@hyperledger-labs/yui-ibc-solidity/IBCPacket");
+const IBCHandler = artifacts.require("@hyperledger-labs/yui-ibc-solidity/OwnableIBCHandler");
 const IBCMsgs = artifacts.require("@hyperledger-labs/yui-ibc-solidity/IBCMsgs");
-const IBCIdentifier = artifacts.require("@hyperledger-labs/yui-ibc-solidity/IBCIdentifier");
 const MockClient = artifacts.require("@hyperledger-labs/yui-ibc-solidity/MockClient");
 
 const PortTransfer = "transfer"
 const MockClientType = "mock-client"
 
 const deployCore = async (deployer) => {
-  await deployer.deploy(IBCIdentifier);
-  await deployer.link(IBCIdentifier, [
-    IBCHost, IBCHandler,
-    MockClient
-  ]);
-
   await deployer.deploy(IBCMsgs);
-  await deployer.link(IBCMsgs, [
-    IBCClient,
-    IBCConnection,
-    IBCChannel,
-    IBCHandler,
-  ]);
+  await deployer.deploy(IBCChannel);
```

Comment:
Can you remove deployments of `IBCMsgs` and `IBCChannel`(=`Channel.sol`), too?

Reply:
Remove IBCMsgs or IBCChannel causes the following error
![スクリーンショット 2023-04-04 16 31 34](https://user-images.githubusercontent.com/126585618/229721679-55f0898c-a76f-4781-b460-2f92ebd18aea.png)

Reply:
The internal functions defined in libraries are compiled into the bytecodes of caller contract.
Since all functions of `Channel.sol` are internal and `IBCMsgs` has even no function definitions, the two libraries need not to be deployed.

Reply:
https://github.com/hyperledger-labs/yui-docs/pull/67/commits/06f5cabb9c3b598c63a8a8886401e0984641a6d1
Thank you. 
IBCHandler's address seems to have changed and TX failed. Changed to correct address.